docs/UserGuide.md

```diff
@@ -359,32 +450,62 @@ Format: `list`
 
 #### Linking a student to a class : `link`
 
-Links an existing student to an existing class in the application.
-
-Format: `link s/STUDENT_INDEX c/CLASS_INDEX`
+You can link a student to a class using this command.
 
+Format: `link s\STUDENT_INDEX c\CLASS_INDEX`
 * Links the student at the specified `STUDENT_INDEX` to the class at the specified `CLASS_INDEX`.
-* `STUDENT_INDEX` refers to the index number shown in the displayed student list.
-* `CLASS_INDEX` refers to the index number shown in the displayed class list.
+* The `STUDENT_INDEX` refers to the index number shown in the displayed student list.
+* The `CLASS_INDEX` refers to the index number shown in the displayed class list.
```

Comment:
Should we put this two terms into glossary? Since they appear in quite a number of places